lib/commands/web-view/set-web-view.ts

```diff
@@ -0,0 +1,58 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+import commandParams = require("../../common/command-params");
+import semver = require("semver");
+
+export class SetWebViewCommand implements ICommand {
+	constructor(private $webViewService: IWebViewService,
+		private $injector: IInjector,
+		private $errors: IErrors,
+		private $logger: ILogger,
+		private $project: Project.IProject) { }
+	
+	public execute(args: string[]): IFuture<void> {
+		return (() => {		
+			this.$webViewService.enableWebView(args[0], args[1]).wait();
+			this.$logger.out("WebView has been successfully changed.");
```

Comment:
Operation completed successfully. Your project now uses the X web view for Y.

X - the web view name
Y - the operating system